apps/design-tokens-ui/src/services/palette/palette-source.service.ts

```diff
@@ -107,9 +104,10 @@ export class PaletteSourceService implements OnDestroy {
 
   constructor(
     router: Router,
-    private _styleOverridesService: StyleOverridesService,
-    @Inject(DOCUMENT) private _document: Document,
+    private _styleOverridesService: StyleOverridesService, // @Inject(DOCUMENT) private _document: Document,
```

Comment:
I think you can delete this
```suggestion
    private _styleOverridesService: StyleOverridesService,
```